packages/webservice/models/build.gradle.kts

```diff
@@ -62,54 +67,45 @@ publishing {
                         email.set("team-navi@googlegroups.com")
                     }
                 }
+                scm {
+                    connection.set("scm:git:https://github.com/yavin-dev/framework.git")
+                    developerConnection.set("scm:git:ssh:git@github.com:yavin-dev/framework.git")
+                    url.set("http://yavin.dev")
+                }
+            }
+        }
+    }
+    repositories {
+        maven {
+            val releasesRepoUrl = uri("https://s01.oss.sonatype.org/service/local/staging/deploy/maven2/")
+            val snapshotsRepoUrl = uri("https://s01.oss.sonatype.org/content/repositories/snapshots/")
+            url = if (project.hasProperty("isSnapshot") && project.property("isSnapshot").toString().equals("true")) snapshotsRepoUrl else releasesRepoUrl
```

Comment:
This line is evaluated before `gradle.taskGraph.whenReady` so the version did not contain "SNAPSHOT" in name so it was using releaseUrl instead of snapshotUrl. introduced a new property isSnapshot to control this.

Reply:
Maybe we should leave a comment in the code for this